Documentation/contributing.rst

```diff
@@ -136,6 +136,8 @@ kubernetes installed and plus a worker, run:
 
 	$ RUNTIME=containerd K8S=1 NWORKERS=1 contrib/vagrant/start.sh
 
+If you want to connect to dev k8s cluster via kubectl from your host machine, set ``KUBECONFIG`` env var to ``$GOPATH/src/github.com/cilium/cilium/vagrant.kubeconfig`` and add ``127.0.0.1 k8s1`` line to your hosts file.
```

Comment:
Avoid using abbreviations in documentation.

"kubectl" --> "\`\`kubectl\`\`\"
"connect to dev k8s cluster" --> "connect to the developer VM provisioned with Kubernetes"
"env var" --> "environment variable"
"line to your hosts file" --> "to your hosts file"

Reply:
Thanks. Changed the "connect" section point out that he is connecting to k8s cluster.